.gitignore

```diff
@@ -0,0 +1,183 @@
+## Ignore Visual Studio temporary files, build results, and
```

Comment:
This is the default `.gitignore` that github generates for you in a new repo if you tick the box and select the `Visual Studio` version in drop down.